[2025-01-15T23:22:14.716+0500] {processor.py:186} INFO - Started process (PID=3746) to work on /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-15T23:22:14.717+0500] {processor.py:914} INFO - Processing file /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-01-15T23:22:14.718+0500] {logging_mixin.py:190} INFO - [2025-01-15T23:22:14.718+0500] {dagbag.py:588} INFO - Filling up the DagBag from /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-15T23:22:14.774+0500] {logging_mixin.py:190} INFO - [2025-01-15T23:22:14.773+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-01-15T23:22:14.775+0500] {logging_mixin.py:190} INFO - [2025-01-15T23:22:14.775+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-01-15T23:22:14.775+0500] {processor.py:927} WARNING - No viable dags retrieved from /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-15T23:22:14.924+0500] {processor.py:208} INFO - Processing /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.214 seconds
[2025-01-15T23:26:37.832+0500] {processor.py:186} INFO - Started process (PID=4907) to work on /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-15T23:26:37.834+0500] {processor.py:914} INFO - Processing file /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-01-15T23:26:37.836+0500] {logging_mixin.py:190} INFO - [2025-01-15T23:26:37.836+0500] {dagbag.py:588} INFO - Filling up the DagBag from /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-15T23:26:37.890+0500] {logging_mixin.py:190} INFO - [2025-01-15T23:26:37.889+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-01-15T23:26:37.908+0500] {logging_mixin.py:190} INFO - [2025-01-15T23:26:37.908+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-01-15T23:26:37.909+0500] {processor.py:927} WARNING - No viable dags retrieved from /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-15T23:26:38.088+0500] {processor.py:208} INFO - Processing /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.261 seconds
[2025-01-15T23:30:32.851+0500] {processor.py:186} INFO - Started process (PID=6162) to work on /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-15T23:30:32.853+0500] {processor.py:914} INFO - Processing file /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-01-15T23:30:32.856+0500] {logging_mixin.py:190} INFO - [2025-01-15T23:30:32.855+0500] {dagbag.py:588} INFO - Filling up the DagBag from /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-15T23:30:32.896+0500] {logging_mixin.py:190} INFO - [2025-01-15T23:30:32.896+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-01-15T23:30:32.896+0500] {logging_mixin.py:190} INFO - [2025-01-15T23:30:32.896+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-01-15T23:30:32.897+0500] {processor.py:927} WARNING - No viable dags retrieved from /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-15T23:30:32.996+0500] {processor.py:208} INFO - Processing /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.149 seconds
[2025-01-15T23:33:34.009+0500] {processor.py:186} INFO - Started process (PID=7016) to work on /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-15T23:33:34.010+0500] {processor.py:914} INFO - Processing file /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-01-15T23:33:34.012+0500] {logging_mixin.py:190} INFO - [2025-01-15T23:33:34.012+0500] {dagbag.py:588} INFO - Filling up the DagBag from /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-15T23:33:34.054+0500] {logging_mixin.py:190} INFO - [2025-01-15T23:33:34.053+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-01-15T23:33:34.054+0500] {logging_mixin.py:190} INFO - [2025-01-15T23:33:34.054+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-01-15T23:33:34.054+0500] {processor.py:927} WARNING - No viable dags retrieved from /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-15T23:33:34.167+0500] {processor.py:208} INFO - Processing /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.163 seconds
[2025-01-15T23:36:27.472+0500] {processor.py:186} INFO - Started process (PID=7764) to work on /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-15T23:36:27.475+0500] {processor.py:914} INFO - Processing file /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-01-15T23:36:27.477+0500] {logging_mixin.py:190} INFO - [2025-01-15T23:36:27.477+0500] {dagbag.py:588} INFO - Filling up the DagBag from /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-15T23:36:27.515+0500] {logging_mixin.py:190} INFO - [2025-01-15T23:36:27.515+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-01-15T23:36:27.516+0500] {logging_mixin.py:190} INFO - [2025-01-15T23:36:27.516+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-01-15T23:36:27.516+0500] {processor.py:927} WARNING - No viable dags retrieved from /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-15T23:36:27.613+0500] {processor.py:208} INFO - Processing /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.146 seconds
[2025-01-15T23:39:10.553+0500] {processor.py:186} INFO - Started process (PID=8533) to work on /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-15T23:39:10.555+0500] {processor.py:914} INFO - Processing file /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-01-15T23:39:10.557+0500] {logging_mixin.py:190} INFO - [2025-01-15T23:39:10.556+0500] {dagbag.py:588} INFO - Filling up the DagBag from /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-15T23:39:10.602+0500] {logging_mixin.py:190} INFO - [2025-01-15T23:39:10.601+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-01-15T23:39:10.603+0500] {logging_mixin.py:190} INFO - [2025-01-15T23:39:10.602+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-01-15T23:39:10.603+0500] {processor.py:927} WARNING - No viable dags retrieved from /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-15T23:39:10.741+0500] {processor.py:208} INFO - Processing /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.193 seconds
[2025-01-15T23:42:29.951+0500] {processor.py:186} INFO - Started process (PID=9303) to work on /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-15T23:42:29.954+0500] {processor.py:914} INFO - Processing file /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-01-15T23:42:29.956+0500] {logging_mixin.py:190} INFO - [2025-01-15T23:42:29.955+0500] {dagbag.py:588} INFO - Filling up the DagBag from /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-15T23:42:29.999+0500] {logging_mixin.py:190} INFO - [2025-01-15T23:42:29.999+0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-01-15T23:42:29.999+0500] {logging_mixin.py:190} INFO - [2025-01-15T23:42:29.999+0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-01-15T23:42:30.000+0500] {processor.py:927} WARNING - No viable dags retrieved from /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-15T23:42:30.107+0500] {processor.py:208} INFO - Processing /Users/mac/Desktop/Comapny2/CryptoPortfolioAI/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.161 seconds
